[docs] update package, features and config mode docs (#3188)
freifunk-gluon/gluon: docs/features/tls.rst

TLS support
===========

The generic TLS implementation which is currently used by OpenWRT can be installed or added as dependency through the package ``gluon-tls``.
This removes the need for community packages to depend on a specific TLS implementation (like mbedtls, OpenSSL or WolfSSL).

> This package is a placeholder for the current TLS implementation used.

Reviewer: ```suggestion
This package is an alias for the current TLS implementation used.
```
